### PR TITLE
add Honeybadger.event interface

### PR DESCRIPTION
**Pull request by mdg**
send application events to the event endpoint
add a timestamp

issue #554
https://github.com/honeybadger-io/honeybadger-elixir/issues/554

### CODE REVIEW

File: lib/honeybadger.ex [hunk 1]
```diff
@@ -332,6 +332,20 @@ defmodule Honeybadger do
...
+    ts = DateTime.utc_now() |> DateTime.to_string()
+    event_data
+    |> Map.put(:event_type, event_type)
+    |> Map.put(:ts, ts)
+    |> Client.send_event()
+  end
+
+  @spec event(map()) :: :ok
+  def event(event_data) do
+    Client.send_event(event_data)
```
Comment (stympy) [inline lib/honeybadger.ex:346]: Could you supply a `ts` of `DateTime.utc_now()` here if one isn't present in the map?

**Review by stympy — APPROVED** [review]
This is looking good, thanks! Could you address the `ts` comment and run `mix format`?

**Comment by mdg** [comment]
@stympy thanks for the helpful prompt review. I think those 2 issues should be addressed now.

**Review by stympy — CHANGES_REQUESTED** [review]
Looking good. I have some [suggested changes](https://github.com/pathpub/honeybadger-elixir/pull/1) for you. :)

**Review by stympy — APPROVED** [review]

**Comment by stympy** [comment]
Released in 0.22.0. Thanks!

**Comment by mdg** [comment]
I've upgraded our app back to the main version. Thanks for handling it so quickly!

**Pull request merged**